src/index.js

```diff
@@ -12,15 +12,15 @@ const root = createRoot(rootElement);
 root.render(
   <React.StrictMode>
     <BrowserRouter> 
-      <Auth0ProviderWithNavigate
+      {/* <Auth0ProviderWithNavigate
         domain={domain}
         clientId={clientId}
         authorizationParams={{
           redirect_uri: "http://localhost:3000/"
         }}
-      >
+      > */}
```

Comment:
i cant wait for us to learn how to get the testing for it to work when we have more time!